internal/template/templates/.claude/agents/moai/manager-develop.md

```
@@ -1,3 +1,4 @@
+isolation: worktree
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)